datadog_lambda/tracing.py

```diff
@@ -412,13 +412,26 @@ def create_inferred_span(event, context):
         elif event_source.equals(EventTypes.SNS):
             logger.debug("SNS event detected. Inferring a span")
             return create_inferred_span_from_sns_event(event, context)
+        elif event_source.equals(EventTypes.KINESIS):
+            logger.debug("Kinesis event detected. Inferring a span")
+            return create_inferred_span_from_kinesis_event(event, context)
+        elif event_source.equals(EventTypes.DYNAMODB):
+            logger.debug("Dynamodb event detected. Inferring a span")
+            return create_inferred_span_from_dynamodb_event(event, context)
+        elif event_source.equals(EventTypes.S3):
+            logger.debug("S3 event detected. Inferring a span")
+            return create_inferred_span_from_s3_event(event, context)
+        elif event_source.equals(EventTypes.EVENTBRIDGE):
+            logger.debug("Eventbridge event detected. Inferring a span")
+            return create_inferred_span_from_eventbridge_event(event, context)
 
     except Exception as e:
         logger.debug(
             "Unable to infer span. Detected type: {}. Reason: {}",
             event_source.to_string(),
             e,
         )
+        print(e)
```

Comment:
do we need this and the logger.debug?

Reply:
Sorry, that sneaked in there with the evenBridge change, ✂️